qhana_plugin_runner/api/plugin_schemas.py

```diff
@@ -52,6 +52,56 @@ class DataMetadataSchema(MaBaseSchema):
     )
 
 
+class ProgressMetadataSchema(MaBaseSchema):
+    value = ma.fields.Integer(
+        required=True, allow_none=False, metadata={"description": "The progress value."}
+    )
+    start = ma.fields.Integer(
+        required=True,
```

Comment:
so just set `required=False`?

Reply:
Yes, but provide a value when it is missing in the model. This value will then be used as the default during serialization or deserialization. But this can probably stay as it is because the default values are already procided elsewhere which would mean that providing them here too is code duplication.